app/assets/javascripts/admin/components/ace-editor.js.es6

```diff
@@ -6,6 +6,8 @@ export default Ember.Component.extend({
   classNames: ['ace-wrapper'],
   _editor: null,
   _skipContentChangeEvent: null,
+  _unusedStashSelf: null,
+  stashSelf: Em.computed.alias('_unusedStashSelf'),
```

Comment:
Why are you stashing self into a property? It seems really odd to me.

Also besides setting it to a  self and later null it seems unused?